superset-frontend/src/messageToasts/components/Toast.tsx

```diff
@@ -76,16 +77,17 @@ export default function Toast({ toast, onCloseToast }: ToastPresenterProps) {
     };
   }, [handleClosePress, toast.duration]);
 
-  let iconName: IconName = 'circle-check-solid';
   let className = 'toast--success';
+  let icon = <Icons.CircleCheckSolid css={theme => StyledIcon(theme)} />;
+
   if (toast.toastType === ToastType.WARNING) {
-    iconName = 'warning-solid';
+    icon = <Icons.WarningSolid css={theme => StyledIcon(theme)} />;
```

Comment:
This should work I think
```suggestion
    icon = <Icons.WarningSolid css={StyledIcon} />;
```